lib/blockscore/base.rb

```diff
@@ -33,12 +33,12 @@ def inspect
 
     def save
       save!
-    rescue BlockScore::Error
+    rescue Error
       false
     end
 
     def save!
-      fail BlockScore::Error, "#{resource} is already deleted" if deleted?
+      assert_not_deleted
       capture_attributes(post(endpoint, attributes))
```

Comment:
There are no specs to make sure the proper attributes are passed to the service.